src/main/java/de/vill/conversion/ConvertGroupCardinality.java

```diff
@@ -81,6 +81,11 @@ private Constraint createConjunction(Set<Feature> selectedFeatures, Set<Feature>
     }
 
     private Constraint createDisjunction(Set<Constraint> constraints) {
+        MultiOrConstraint orConstraint = new MultiOrConstraint();
+        for (Constraint constraint : constraints) {
+            orConstraint.add_sub_part(constraint);
+        }
+        /*
```

Comment:
Needed?

Reply:
No, I guess not. Thats the original code before introducing MultiOrs. If we are fine with keeping the MultiOr, I can remove it.